src/providers/idp/idp_auth.c

```diff
@@ -0,0 +1,543 @@
+/*
+    SSSD
+
+    IdP Backend Module -- Authentication
+
+    Authors:
+        Sumit Bose <sbose@redhat.com>
+
+    Copyright (C) 2024 Red Hat
```

Comment:
2025? Same on other places, you even have 2009 somewhere :-)

Reply:
Hi,

most of the code was already available 2024, so I think this is ok, but I fixed the 2009.

bye,
Sumit